### DISCUSSION TITLE
Amazon Sensor Issues

### DISCUSSION THREAD

**Discussion by dfoley4** [moralmunky/Home-Assistant-Mail-And-Packages#586] [Q&A]
Hello,

I've had success with all the sensors but the Amazon one. I suspect it has something to do with the forwarding email since that's the only difference. I'm running the latest beta (0.3.4b9) and also had the same issues on the latest stable build (0.3.3-2).

I've got a forwarding rule from Gmail to Gmail and have the forwarding email set in the config but it isn't picking up any of the numerous packages that are currently en route.

Any tips on how to ensure the integration is picking up Amazon packages?

**Comment by firstof9** [comment]
Packages only get counted on the day of their delivery. This is [covered in the wiki](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki#about-mail-and-packages-integration).

**Comment by dfoley4** [comment]
Thanks for your reply, I do see that the other package sensors only show on the day of delivery but even during Amazon delivery days the sensor shows 0. I double-checked and the emails are being forwarded correctly but the sensor doesn't seem to pick them up. Any ideas on what to look into?

**Comment by firstof9** [comment]
This can happen if you delete the Amazon email before the delivery occurs.
